int64 request_number
string request_type
int16 command
